src/interactive-window/interactiveWindow.ts

```diff
@@ -78,11 +75,6 @@ export function getInteractiveCellMetadata(cell: NotebookCell): InteractiveCellM
     }
 }
 export class InteractiveWindow implements IInteractiveWindowLoadable {
-    public get ready() {
```

Comment:
Remove all of this, its much simpler now.
Relied on the factory class to create and provide all of he dependencies that wa y we don't end up with weird promises & nullable types.